### DISCUSSION TITLE
Qual a diferença entre um aplicativo web e um aplicativo desktop criado com o Electron?

### DISCUSSION THREAD

**Discussion by KAYOKG** [KAYOKG/TCC-beta-version#20] [Q&A] [answered]
Qual a diferença entre um aplicativo web e um aplicativo desktop criado com o Electron?

**Comment by SolarisSy** [comment, marked as answer]
Um aplicativo web é um software que pode ser acessado através de um navegador da web, enquanto um aplicativo desktop é instalado diretamente no sistema operacional do computador do usuário.

O Electron permite que aplicativos web sejam empacotados como aplicativos desktop, tornando-os disponíveis para uso offline e oferecendo recursos adicionais, como acesso ao sistema de arquivos e integração com outras aplicações do sistema operacional.

Além disso, a criação de um aplicativo desktop com o Electron e a stack MERN permite utilizar tecnologias web familiares para criar aplicativos desktop que se integram com bancos de dados e outras aplicações de desktop.
